res/css/views/settings/tabs/user/_LabsUserSettingsTab.scss

```diff
@@ -20,6 +20,9 @@ limitations under the License.
 
         .mx_SettingsFlag {
             margin-right: 0; // remove right margin to align with beta cards
+            display: flex;
+            align-items: center;
+            justify-content: space-between;
         }
```

Comment:
```suggestion
        .mx_SettingsFlag {
            margin-right: 0; // remove right margin to align with beta cards
            display: flex;
            align-items: center;
            justify-content: space-between;
            
            .mx_ToggleSwitch {
                float: unset;
            }
        }
```

Reply:
the float unset is just to clear the float: right as now we're using flexbox + justify-content: space-between for it